extra/1-currency-conversion.js

```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(value) {
+  var convert = value*5.7*0.99;
+  return parseFloat(convert.toFixed(2));
```

Comment:
You could assign the different numbers to variables with meaningful names to make the code more understandable, i.e. USDExchangeRate. This will also make the values easier to change in future.

Reply:
Done